CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Note: The `kubernetes:helm/v2:Chart` API was deprecated in this update and will
 updates.
 
 - Deprecate helm/v2:Chart resources (https://github.com/pulumi/pulumi-kubernetes/pull/1990)
+- Don't use the last-applied-configuration annotation for CRDs (https://github.com/pulumi/pulumi-kubernetes/pull/1882)
```

Comment:
Can we verify if this obviates the enableReplaceCRD flag? In which case lets remove and call it out explicitly in the changelog.

Reply:
Good catch! I verified that the `enableReplaceCRD` option is no longer required, and added a deprecation notice to clarify that it is no longer used and will be removed in the future.